src/commands/init_dataset.rs

```diff
@@ -6,21 +6,22 @@ use directories::ProjectDirs;
 use super::InitDatasetArgs;
 
 pub(crate) fn init_dataset(args: &InitDatasetArgs, config: &mut E4EDMConfig) -> Result<()> {
+    let manifest_root = args.path.to_owned()
+        .unwrap_or(ProjectDirs::from("com", "Engineers For Exploration", "E4EDataManagement").unwrap().config_dir().to_path_buf());
```

Comment:
@ntlhui Do we care about qualifiers like this?

Reply:
Per https://yancouto.github.io/functional/directories/struct.ProjectDirs.html#examples, this is probably only relevant for MacOS.  Strictly, the qualifier should be `["edu", "UCSD", "Engineers for Exploration", "E4EDataManagement"]` if we chose to use one at all.